doc/classes/XRFaceTracker.xml

```diff
@@ -0,0 +1,469 @@
+			<param index="0" name="blend_shape" type="int" enum="XRFaceTracker.BlendShapeEntry" />
+			<param index="1" name="weight" type="float" />
+			<description>
+				Sets a face blend shape weight.
+			</description>
+		</method>
+	</methods>
+	<members>
+		<member name="blend_shapes" type="PackedFloat32Array" setter="set_blend_shapes" getter="get_blend_shapes" default="PackedFloat32Array(0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0)">
+			The array of face blend shape weights.
+		</member>
+	</members>
+	<constants>
+		<constant name="FT_EYE_LOOK_OUT_RIGHT" value="0" enum="BlendShapeEntry">
+			Right eye looks outwards.
+		</constant>
+		<constant name="FT_EYE_LOOK_IN_RIGHT" value="1" enum="BlendShapeEntry">
+			Right eye looks inwards.
+		</constant>
+		<constant name="FT_EYE_LOOK_UP_RIGHT" value="2" enum="BlendShapeEntry">
+			Right eye looks upwards.
+		</constant>
+		<constant name="FT_EYE_LOOK_DOWN_RIGHT" value="3" enum="BlendShapeEntry">
+			Right eye looks downwards.
+		</constant>
+		<constant name="FT_EYE_LOOK_OUT_LEFT" value="4" enum="BlendShapeEntry">
+			Left eye looks outwards.
+		</constant>
+		<constant name="FT_EYE_LOOK_IN_LEFT" value="5" enum="BlendShapeEntry">
+			Left eye looks inwards.
+		</constant>
+		<constant name="FT_EYE_LOOK_UP_LEFT" value="6" enum="BlendShapeEntry">
+			Left eye looks upwards.
+		</constant>
+		<constant name="FT_EYE_LOOK_DOWN_LEFT" value="7" enum="BlendShapeEntry">
+			Left eye looks downwards.
+		</constant>
+		<constant name="FT_EYE_CLOSED_RIGHT" value="8" enum="BlendShapeEntry">
+			Closes the right eyelid.
+		</constant>
+		<constant name="FT_EYE_CLOSED_LEFT" value="9" enum="BlendShapeEntry">
+			Closes the left eyelid.
+		</constant>
+		<constant name="FT_EYE_SQUINT_RIGHT" value="10" enum="BlendShapeEntry">
+			Squeezes the right eye socket muscles.
+		</constant>
+		<constant name="FT_EYE_SQUINT_LEFT" value="11" enum="BlendShapeEntry">
+			Squeezes the left eye socket muscles.
+		</constant>
+		<constant name="FT_EYE_WIDE_RIGHT" value="12" enum="BlendShapeEntry">
+			Right eyelid widens beyond relaxed.
+		</constant>
+		<constant name="FT_EYE_WIDE_LEFT" value="13" enum="BlendShapeEntry">
+			Left eyelid widens beyond relaxed.
+		</constant>
+		<constant name="FT_EYE_DILATION_RIGHT" value="14" enum="BlendShapeEntry">
+			Dilates the right eye pupil.
+		</constant>
+		<constant name="FT_EYE_DILATION_LEFT" value="15" enum="BlendShapeEntry">
+			Dilates the left eye pupil.
+		</constant>
+		<constant name="FT_EYE_CONSTRICT_RIGHT" value="16" enum="BlendShapeEntry">
+			Constricts the right eye pupil.
```

Comment:
May be more accurate but I *did* have to look up what it meant as a non-native English speaker.
```suggestion
			Contracts the right eye pupil.
```

Reply:
The names and descriptions come from the [Unified Expressions](https://docs.vrcft.io/docs/tutorial-avatars/tutorial-avatars-extras/unified-blendshapes) standard. They actually put pictures showing the side-effect on meshes because facial distortions and the associated terminology are a specialized subject.

That website does have hover-over popups for each blend-shape with corresponding anatomical information and enhanced descriptions. Perhaps using the extended descriptions may be more useful.

Reply:
Being more detailed here, since there are no images would definitely be useful. But I also do believe, wherever the description comes from, it can be "toned down" a bit for most readers.

Reply:
@Mickeon I'm kinda with Malcolm on this one. The descriptions come directly from the standard and we refer to the standard. I don't think we should be re-inventing the wheel here.

People who work on this stuff will be looking at the unified expression standard and documentation around this, or be already familiar with the standard. If we suddenly deviate that is only going to lead to extra confusion.

Getting people up and running who aren't familiar here, there is a lot more information to disseminate. including a lot of stuff around modeling and how to setup the blend shapes in the first place. That is more tutorial grounds and could use a nice step by step write down in the manual, or a good video tutorial.

Reply:
The standard is maintained in Github at https://github.com/VRCFaceTracking/docs and the author has requested feedback and PR improvements. Possibly feeding improvements through the standard might be the better approach as Godot is adopting this standard.

I am concerned about creating too many URL links to the standard as it would put us at the mercy of URL path changes; and I'm not sure we have the right to copy the images.